lnbits/wallets/fake.py

```diff
@@ -78,23 +77,21 @@ async def create_invoice(
 
         return InvoiceResponse(True, checking_id, payment_request)
 
-    async def pay_invoice(self, bolt11: str, fee_limit_msat: int) -> PaymentResponse:
+    async def pay_invoice(self, bolt11: str, _: int) -> PaymentResponse:
         invoice = decode(bolt11)
-        if (
-            hasattr(invoice, "checking_id")
-            and invoice.checking_id[:6] == self.privkey[:6]  # type: ignore
-        ):
+
+        if invoice.payment_hash[:6] == self.privkey[:6]:
```

Comment:
- is this change related/required by `pyright` (use `payment_hash` instead of `checking_id`)?